### PR TITLE
Update documentation for packages built using python wheels

### PR DESCRIPTION
**Pull request by matt-chan**
The tests aren't included on packages build in PyPi. Updating documentation to show that.

### CODE REVIEW

File: master/docs/manual/installation/installation.rst [hunk 1]
```diff
@@ -75,7 +75,9 @@ If you cannot or do not wish to install the buildbot into a site-wide location l
 Running Buildbot's Tests (optional)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you wish, you can run the buildbot unit test suite.
+If you wish, you can run the buildbot unit test suite. You must not be using a python wheels packaged version
```
Comment (sa2ajj) [inline master/docs/manual/installation/installation.rst]: Buildbot is spelled as `Buildbot`.

We have "one sentence per line policy" (http://docs.buildbot.net/latest/developer/style.html)

**Comment by matt-chan** [comment]
Sorry about that. I didn't realize about the one-line thing and the capitalization. Fixed to reflect that now.

Also rebased to follow master again.

File: master/docs/manual/installation/installation.rst [hunk 2]
```diff
@@ -75,7 +75,9 @@ If you cannot or do not wish to install the buildbot into a site-wide location l
 Running Buildbot's Tests (optional)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you wish, you can run the buildbot unit test suite.
+If you wish, you can run the Buildbot unit test suite. 
+You must not be using a python wheels packaged version of Buildbot or have specified the bdist_wheel flag when building.
```
Comment (sa2ajj) [inline master/docs/manual/installation/installation.rst]: Python is spelled Python.

Comment (sa2ajj) [inline master/docs/manual/installation/installation.rst]: I believe `bdist_wheel` is a command, not a flag.

I think an example of the command line with `bdist_wheel` would be great.

**Comment by sa2ajj** [comment]
Otherwise, LGTM

**Comment by matt-chan** [comment]
Rebased to follow master. Also applied changes discussed. Thanks!

**Comment by sa2ajj** [comment]
Looks like it's still can't be merged to `master`.  Could you please rebase it once more?

**Pull request merged**